### PR TITLE
Tinker writer

### PR DESCRIPTION
**Pull request by micaela-matta**
Fixes #

Changes made in this Pull Request:

 - added Tinker trajectory writer 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by jbarnoud** [comment]
Thank you for the pull request. I do not have time to look at it right now, but I'll have a look as soon as possible if nobody reviewed the PR in the meantime.

For the tests to run, you need to add `new_hexane.arc` in `MDAnalysisTests.datafiles` as `ARC_PBC`. I suggest you run the tests locally to catch this kind of errors without waiting for travis. The wiki documents how to run the tests: <https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests>.

**Comment by jbarnoud** [comment]
I did not see it was related to #2011.

**Comment by micaela-matta** [comment]
Thanks for pointing that out. Maybe after I've corrected that @richardjgowers  can have a look.

File: package/MDAnalysis/topology/TXYZParser.py [hunk 1]
```diff
@@ -88,9 +89,17 @@ def parse(self, **kwargs):
             names = np.zeros(natoms, dtype=object)
             types = np.zeros(natoms, dtype=np.int)
             bonds = []
+            #Checks whether file contains periodic boundary box info 
+            fline = inf.readline()
+            try:
+                float(fline.split()[1])
+            except:
```
Comment (tylerjereddy) [inline package/MDAnalysis/topology/TXYZParser.py]: Catching all exception classes, including keyboard interruptions, might not be desired

**Comment by tylerjereddy** [comment]
> I did not see it was related to #2011.

Indeed, clarifying if this PR is still relevant / needed might be useful given period of inactivity.

**Comment by hmacdope** [comment]
Closing as stale, feel free to re-open if you want to continue.

**Pull request closed without merging**